src/components/ActivityFeed/ActivityFeedItem.js

```diff
@@ -735,16 +738,16 @@ export class ActivityFeedItem extends React.Component<Props> {
               noSymbol: !formattedFullValue,
             }),
             itemValue: getFormattedValue(formattedValue, event.asset, { isPositive: isReceived, noSymbol: isZero }),
-            valueColor: isReceived && !this.isZeroValue(value) ? 'positive' : 'text',
+            valueColor: isReceived && !this.isZeroValue(value) ? 'secondaryAccent140' : 'basic010',
             ...additionalInfo,
             isReceived,
           };
         }
     }
     data.itemStatusIcon = data.itemStatusIcon || (isPending ? TX_PENDING_STATUS : '');
     if (isFailed) {
-      data.itemStatusIcon = TX_FAILED_STATUS;
-      data.statusIconColor = this.getColor('negative');
+      data.itemStatusIcon = isPPNTransaction ? '' : TX_FAILED_STATUS;
```

Comment:
Wdyt? As well maybe you remember why we have this check, it would be to have it as note here
```suggestion
      if (isPPNTransaction) data.itemStatusIcon = TX_FAILED_STATUS;
```

Reply:
👍 
Will update, thanks!

As per check - it's based on different error styles for PPN and regular failed transactions.
Regular failed:
<img width="563" alt="regular_failed" src="https://user-images.githubusercontent.com/17533987/99044003-bf050a00-2597-11eb-8358-c8c4a75cb196.png">

PPN failed:
<img width="565" alt="ppn_failed" src="https://user-images.githubusercontent.com/17533987/99044015-c3c9be00-2597-11eb-8dcb-402efd3c332a.png">

[source](https://app.zeplin.io/project/5db9b5d14c30ae56faaa3030/screen/5e70acc46caa6013a7165a2e)

Will add a note :)